.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-    "stage": 2
+  "presets": ["react", "stage-2"],
```

Comment:
I'd normally just set the whole `es2015` prefix, but use only a subset of features.

Do we still need stage 2?